recipe/meta.yaml

```
@@ -10,35 +10,45 @@ source:
   sha256: bfa03788803061c80c005e28fab5d96f465fcf1c7d28a1eefb1631fb4a2ab67f
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - setuptools
+    - wheel
   run:
-    - python >=3.7
+    - python
     - google-api-core >=1.34.1,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
```

```suggestion
    - google-api-core >=1.34.1,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
    - grpcio >=1.33.2,< 2.0dev
    - grpcio >= 1.49.1,< 2.0dev # [py>=311]
    - grpcio-status >=1.33.2,< 2.0dev
    - grpcio-status >= 1.49.1,< 2.0dev # [py>=311]
```
upstreams requires google-api-core[grpc] https://github.com/googleapis/google-cloud-python/blob/google-ai-generativelanguage-v0.6.12/packages/google-ai-generativelanguage/setup.py#L44C6-L44C21

There is no good way to express this. We can however refer to https://github.com/googleapis/google-cloud-python/blob/google-ai-generativelanguage-v0.6.12/packages/google-ai-generativelanguage/setup.py#L44C6-L44C21 and https://inspector.pypi.io/project/google-api-core/2.23.0/packages/fa/6b/b98553c2061c4e2186f5bbfb1aa1a6ef13fc0775c096d18595d3c99ba023/google_api_core-2.23.0.tar.gz/google_api_core-2.23.0/pyproject.toml#line.62